__tests__/app.test.js

```diff
@@ -222,6 +222,54 @@ describe("/api", () => {
                     })
                 })
             })
+            describe("Patch", () => {
+                test("Patch /api/articles/:article_id should increase/decrease the vote by a specfied amount on a specified article", () => {
+                    return request(app)
```

Comment:
Your test descriptions applies that it will be testing for both an increase in votes and a decrease. You assertion is only test for an increase so is a bit misleading